Python/Product/Profiling/PythonProfilingPackage.cs

```diff
@@ -361,10 +310,71 @@ protected override int CreateToolWindow(ref Guid toolWindowType, int id)
             config.ScriptArguments = runTarget.Arguments;
             config.WorkingDirectory = runTarget.WorkingDirectory;
 
-            RunProfiler(session, config, openReport);
+            if (useVTune) {
+                RunVTune(session, config, openReport);
+            } else {
+                RunProfiler(session, config, openReport);
+            }
         }
 
 
+        private static void RunVTune(SessionNode session, LaunchConfiguration config, bool openReport) {
+#if false
+            var pyexe = ProcessOutput.QuoteSingleArgument(stndTarget.InterpreterPath);
+
+            if (pyexe == string.Empty) {
+                if (stndTarget.PythonInterpreter != null) {
+                    var registry = session._serviceProvider.GetComponentModel().GetService<IInterpreterRegistryService>();
+                    var interpreter = registry.FindConfiguration(runTarget.PythonInterpreter.Id);
+                    if (interpreter == null) { 
+                        /* ??? */;
+                        MessageBox.Show("Could not find interpreter in the registry");
+                    } else {
+                        MessageBox.Show($"The Python interpreter in question is: [{interpreter}]");
+                    }
+                }
+            }
+#endif
+
+            var interpreter = config.GetInterpreterPath();
+            if (!File.Exists(interpreter)) {
+                MessageBox.Show("Can't find specified python interpreter.");
+                return;
+            }
+
+            string outPathDir = Path.GetTempPath();
+            string outPath = Path.Combine(outPathDir, "pythontrace.diagsession");
```

Comment:
The pythontrace.diagsession and sample.dwjson below will be the exact same path for all visual studio instances. I think it needs to use a random tmp subfolder.